week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,8 +12,26 @@
  */
 
 class ATM {
-    // Add your code here
-
+  constructor() {
+    this.amount = 100.0;
+  }
+
+  make_deposit(amount) {
+    this.amount += amount;
+  }
+
+  check_balance() {
+    console.log("Balance : " + this.amount);
+  }
+
+  make_withdrawl(amount) {
+    // I assume the input amount is in number format
+    if (amount > this.amount) {
+      console.log("Sorry, the balance is not enough");
```

Comment:
Nice check!

Notice this doesn't allow you to withdraw 100 if you have exactly 100 in the bank 👀